code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -259,7 +259,9 @@ GLOBAL_LIST_EMPTY(PDAs)
 				dat += text("<br><a href='?src=[REF(src)];choice=UpdateInfo'>[id ? "Update PDA Info" : ""]</A><br><br>")
 
 				dat += "[station_time_timestamp()]<br>" //:[world.time / 100 % 6][world.time / 100 % 10]"
-				dat += "[time2text(world.realtime, "MMM DD")] [GLOB.year_integer+540]"
+				dat += "[time2text(world.realtime, "MMM DD")] [GLOB.year_integer+540]<br>"
+				dat += "It has been [ROUND_TIME] since the emergency shuttle was last called."
```

Comment:
I feel like this wording is confusing. If I call the emergency shuttle and it get's recalled, does ROUND_TIME get reset to 0?

Reply:
Dammit coiax!!!!

Reply:
I kinda wanted to do "we have gone ROUND_TIME since last incident"

Reply:
I got that vibe, but TBH round time is pretty useful for organizing events ingame, so it's probably best to have the PDA message be as clear as possible, and fit in a joke like that somewhere else. 

If you wanted to for a later PR, a sign on the SM like that would be funny, especially if you read/added the time-of-last-delam to the DB for cross-round fun!

Reply:
Oh shit, I was joking, I just had one of those "X DAYS SINCE LAST ACCIDENT" signs. It's not a requirement at all. 😅